### ISSUE TITLE
Trouble installing on Mac

### ISSUE THREAD

**Issue by wrathagom** [NerdyPepper/dijo#19]
I believe I followed the instructions correctly, but the install ended with:

```
   Compiling dijo v0.1.2
error[E0432]: unresolved import `notify::INotifyWatcher`
 --> /Users/username/.cargo/registry/src/github.com-1ecc6299db9ec823/dijo-0.1.2/src/app/mod.rs:4:30
  |
4 | use notify::{DebouncedEvent, INotifyWatcher};
  |                              ^^^^^^^^^^^^^^ no `INotifyWatcher` in the root

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: failed to compile `dijo v0.1.2`, intermediate artifacts can be found at `/var/folders/gq/6g5prsts0knc_vlvwfhz63mr0000gn/T/cargo-installOYUMfV`

Caused by:
  could not compile `dijo`.
```

**Comment by NerdyPepper** [comment]
The fix for this issue was merged and released earlier today (dijo >= v0.1.3) , could try updating with:

```shell
# if you installed from source
$ git pull origin master
$ cargo install --path . --force

# if you installed from crates.io
$ cargo install dijo
```

**Comment by wrathagom** [comment]
This did get me up and running, thank you very much for the support!
